core/src/main/scala/kafka/server/FetchDataInfo.scala

```diff
@@ -19,4 +19,5 @@ package kafka.server
 
 import kafka.message.MessageSet
 
-case class FetchDataInfo(fetchOffsetMetadata: LogOffsetMetadata, messageSet: MessageSet)
+case class FetchDataInfo(fetchOffsetMetadata: LogOffsetMetadata, messageSet: MessageSet,
+                         messageSetIncomplete: Boolean = false)
```

Comment:
Agreed, this also came up in a conversation between Jason and myself and I was thinking of renaming it along the lines you suggest. :)